niri-ipc/src/lib.rs

```diff
@@ -133,6 +135,8 @@ pub enum Response {
     FocusedWindow(Option<Window>),
     /// Information about the picked window.
     PickedWindow(Option<Window>),
+    /// Information about the picked color as RGBA bytes.
+    PickedColor(Option<[u8; 4]>),
```

Comment:
The GNOME D-Bus interface has the right idea here to use floats, because with 10-bit color depth a u8 is no longer enough. I would also make this an `Option<SomeNewStruct>` because when HDR happens we will need to signal some extra metadata like the color space, and we want to do that without breaking backwards compatibility.